### PR TITLE
Overload vest for convenience

### PR DESCRIPTION
**Pull request by brianmcmichael**
Enables overloaded vest

### CODE REVIEW

**Comment by gbalabasquer** [comment]
`MAX_UINT` assignment should be removed and the base PR also modifies it in the echidna test, so that could be changed back as well.

**Review by gbalabasquer — APPROVED** [review]
LGTM

File: src/DssVest.t.sol [hunk 1]
```diff
@@ -116,7 +115,7 @@ contract DssVestTest is DSTest {
 
         hevm.warp(now + 70 days);
 
-        vest.vest(id, MAX_UINT);
+        vest.vest(id, uint256(-1));
```
Comment (talbaneth) [inline src/DssVest.t.sol:118]: Did you make this one call my overlaoaded function on purpose? It's just different than the other.

**Review by talbaneth — COMMENTED** [review]
see one minor comment in the next review, other than that looks good.

File: src/DssVest.t.sol (hunk 1 above)
Comment (talbaneth) [inline src/DssVest.t.sol:118]: Is this one still using my function on purpose? It's different than the others.

**Review by talbaneth — COMMENTED** [review]
minor comment

**Review by hexonaut — APPROVED** [review]
LGTM

File: src/DssVest.t.sol (hunk 1 above)
Comment (brianmcmichael) [reply src/DssVest.t.sol:118]: yeah. There are two external vest functions now. This test uses them both.

**Review by talbaneth — APPROVED** [review]
LGTM

**Pull request merged**